internal/pkg/mapper/defaultbucket/defaultbucket.go

```diff
@@ -57,8 +59,17 @@ func (m *defaultBucketMapper) visitStorageInputTables(config configOrRow, conten
 	return nil
 }
 
-func markConfigUsedInInputMapping(omConfig *model.Config, imConfig configOrRow) {
-	omConfig.Relations.Add(&model.UsedInInputMappingRelation{
-		ConfigKey: imConfig.Key(),
-	})
+func markUsedInInputMapping(omConfig *model.Config, usedIn configOrRow) {
+	switch v := usedIn.(type) {
+	case *model.Config:
+		omConfig.Relations.Add(&model.UsedInConfigInputMappingRelation{
+			UsedIn: v.ConfigKey,
+		})
+	case *model.ConfigRow:
+		omConfig.Relations.Add(&model.UsedInRowInputMappingRelation{
+			UsedIn: v.ConfigRowKey,
+		})
```

Comment:
Rovno som to pushol, chcel som overit, ze tam nie je ziadny zasek.

Najlepsie riesenie je IMHO mat 2 relacie:
- je to najdlhsie, co sa tyka kodu
- ale nie je v tom ziadna tazko pochopitelna "magia"
- nestane sa, ze by niekto ulozil do `Key` BnrachKey (ako v sucasnom rieseni)
- ani sa nestane, ze by niekto vyplnil aj ConfigKey aj RowKey
- takze takto je to typovo "ciste"

Reply:
vid https://github.com/keboola/keboola-as-code/pull/390/commits/25e0b78031e20df7f6b3037180acdf52cc044917

Reply:
Aha, super. Díky!